aeneas/src/x86/X86MacroAssembler.v3

```diff
@@ -320,6 +411,204 @@ class X86MacroAssembler extends X86Assembler {
 		}
 		if (frame != null) mach.runtime.src.recordFrameEnd(codeOffset());
 	}
+
+	def cast_f2ui(dest: X86Reg, a: SSERm) {
```

Comment:
Let's avoid the name "cast" here to prevent confusion with source- and IR- level cast operations, which can throw exceptions.